src/main/java/apoc/diff/Diff.java

```diff
@@ -0,0 +1,110 @@
+package apoc.diff;
+
+import apoc.Description;
+import apoc.result.DiffResult;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.procedure.*;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author Benjamin Clauss
+ * @since 29.03.2018
+ */
+public class Diff {
+
+    @Context
+    public GraphDatabaseService db;
+
+    @UserFunction
+    @Description("apoc.diff.maps([leftMap],[rightMap])" )
+    public boolean maps(@Name("leftMap") Map<String, Object> leftMap, @Name("rightMap") Map<String, Object> rightMap) {
+        return !leftMap.equals(rightMap);
+    }
+
+    @UserFunction
+    @Description("apoc.diff.properties([leftNodeId],[rightNodeId])")
+    public boolean properties(@Name("leftNodeId") Long leftNodeId, @Name("rightNodeId") Long rightNodeId) {
```

Comment:
support nodes and not node-ids, if someone has node-ids they can use cypher to look them up.
as you want to use this from cypher you most often have nodes, and not node ids.